### PR TITLE
Loggly

### PR DESCRIPTION
**Pull request by AlexChaseJones**

### CODE REVIEW

File: src/scripts/loggly.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      json: true,
+      tags: [GLOBAL_VARS.logglyTag]
+    });
+  }
+
+  setDevice(id) {
+    this.device_id = id;
+  }
+
+  log(method, args) {
```
Comment (dagifted1) [inline src/scripts/loggly.js]: @AlexChaseJones clean this up if possible

**Comment by dagifted1** [comment]
- merge buildkite folder into branch so checks pass.
- write tests for loggly and fix tests

**Comment by AlexChaseJones** [comment]
@dagifted1 loggly module is updated and tests are written. Buildkite is still failing for some reason

File: test/test.js [hunk 2]
```diff
@@ -1,10 +1,62 @@
...
+	const expected = 'MN-12-345567-R'
+	t.is(actual, expected)
+})
+
+test('Loggly throws syntax error', t => {
+	const error = loggly.log('notALifecycleMethod');
+	t.is(error.message, "Loggly.log only accepts a custom object, 'setData', 'render', or 'updateView'")
+})
+
+test('Loggly logger handles API error', t => {
```
Comment (dagifted1) [inline test/test.js]: @AlexChaseJones mock this route in the test and then we should be good.

**Pull request merged**